src/integrations/Mixpanel/browser.js

```diff
@@ -376,6 +380,13 @@ class Mixpanel {
    */
   alias(rudderElement) {
     logger.debug('in Mixpanel alias');
+    if (this.identityMergeApi === 'simplified') {
+      logger.debug(
+        "===Mixpanel: Event type 'alias' is not supported when 'Simplified ID merge' api is selected in webapp===",
```

Comment:
https://docs.mixpanel.com/docs/tracking/how-tos/identifying-users#how-does-the-simplified-api-differ-from-the-the-original-api This doesn't say it is not supported, it is just saying alias, identifies,  merge behaves same, is my understand correct?

Reply:
alias call is nothing but $create_alias event. Which is deprecated and will ignored by mixpanel.
We can improve the log saying that
`"===Mixpanel: Alias call is deprecated in 'Simplified ID Merge'==="`

Reply:
share documentation about deprecation?

Reply:
In the above document you shared. This is what mixpanel has mentioned about the alias call
`In the Simplified API the $identify, $merge, and $create_alias events no longer have any special meaning and will be ignored`

Reply:
it is also talking about $identify too right so that is also not supported? may be we are understand this wrongly?

Reply:
for now it is not not clear from docs but based on testing identify is required